src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (people) => people.reduce(
```

Comment:
It is recommended to use more meaningful function names, consider renaming 'calculateAverageAge' to 'calculateAverageAgeOfPeople'.